### PR TITLE
Calculate proper msg id commands from relayer that have integer overf…

### PR DESCRIPTION
**Pull request by norkans7**
…low issue

### CODE REVIEW

**Comment by nicpottier** [comment]
I thought the problem was that with the updated client it would see now the full long ID and think it hadn't sent it? I guess this addresses one side of it in that we will mark messages on old clients as sent on the RapidPro side but does it address an updated client not getting the new long id and thinking it wasn't sent yet?

File: temba/channels/tests.py [hunk 1]
```diff
@@ -1312,6 +1313,7 @@ def test_sync(self, mr_mocks):
             dict(cmd="mt_dlvd", msg_id=msg3.pk, ts=date),
             dict(cmd="mt_error", msg_id=msg4.pk, ts=date),
             dict(cmd="mt_fail", msg_id=msg5.pk, ts=date),
+            dict(cmd="mt_fail", msg_id=(4294967296 - msg6.pk), ts=date),  # simulate a negative integer from relayer
```
Comment (rowanseymour) [inline temba/channels/tests.py:1316]: is `4294967296 - msg6.pk` negative?

Comment (norkans7) [reply temba/channels/tests.py:1316]: this is a mistake, fixing that

**Comment by norkans7** [comment]
The new client will not have any issue with long integer as those will have been from the server so, actually I think these changes would fix the issue even without the client update for the time being

And it is better we push this and release the new client like a few days after

**Comment by nicpottier** [comment]
Ya, I guess if this goes out first then RapidPro won't ask for them to be sent again. Ok.

**Review by nicpottier — APPROVED** [review]

**Pull request merged**